command.go

```diff
@@ -25,6 +25,7 @@ type Cmder interface {
 
 	readTimeout() *time.Duration
 	readReply(rd *proto.Reader) error
+	readRawReply(rd *proto.Reader) error
```

Comment:
Do you think it might be possible, during readReply, prior to the parsing to store the RAW resp - based on user input of redis.Options? My thinking (maybe crazy) is that:

1. If we have readReply store the raw prior to parsing, it means it's optionally parsed - so it's now faster.
2. If we store by default we dramatically increase memory footprint (I assume x2) per command executed - so we want to avoid that.
3. If it's an optional redis argument of some sort, we make it user possible, without impact.

WDYT?